Formula/libpulsar.rb

```diff
@@ -0,0 +1,44 @@
+class Libpulsar < Formula
+  desc "Apache Pulsar C++ library"
+  homepage "https://pulsar.apache.org"
+
+  url "https://www.apache.org/dyn/mirrors/mirrors.cgi?action=download&filename=pulsar/pulsar-2.2.0/apache-pulsar-2.2.0-src.tar.gz"
+  sha256 "a3b1940a803043bb2c365ce9df657d15bf9aacb3c9ff5db86a79dc4374033f08"
+
+  depends_on "boost-python" => :build
+  depends_on "cmake" => :build
+  depends_on "jsoncpp" => :build
+  depends_on "openssl" => :build
+  depends_on "pkg-config" => :build
+  depends_on "protobuf@2.6" => :build
+
+  depends_on "boost"
+
+  def install
+    cd "pulsar-client-cpp" do
+      system "cmake", ".", "-DBUILD_TESTS=OFF", "-DLINK_STATIC=ON",
+          "-DPYTHON_INCLUDE_DIR=#{Formula["python"].include}",
+          "-DBoost_INCLUDE_DIRS=#{Formula["boost"].include}"
+      system "make", "pulsarShared", "pulsarStatic"
+
+      include.install "include/pulsar"
```

Comment:
`make install` please

Reply:
Problem with current make install is that it installs all the CMake targets, including examples, tests and python bindings which are not meant to be distributed in this case. 

To have a suitable `make install` we'll have to fix the CMakeList.txt in some forms.

Reply:
Actually, I was wrong (I forgot that I wrote the install target to just install the libs and headers). So, the install works fine, the only problem is that it forces the build the python binding and tests that require some additional dependencies. I'm trying to fix the recipe to work with that.